core/tests/test_apps.solana.offchain_message.py

```
@@ -0,0 +1,197 @@
+        ],
+        "message": "Multi-party agreement.",
+        "format": 0,
+        "hex": "ff736f6c616e61206f6666636861696e00054a535a992921064d24e87160da387c7c35b5ddbc92bb81e41fa8404105448d000300000000000000000000000000000000000000000000000000000000000000000761481d357474bb7c4d7624ebd3bdb3d8355e73d11043fc0da353800000000006a1d8179137542a983437bdfe2a7ab2557f535c8a78722b68a49dc00000000016004d756c74692d70617274792061677265656d656e742e",
+    },
+]
+
+
+@unittest.skipUnless(not utils.BITCOIN_ONLY, "altcoin")
+class TestMessageV0(unittest.TestCase):
+
+    def assertMsgEqual(self, offchain, app, signers, format, message):
```

Ruff (0.15.13)

[error] Function argument `format` is shadowing a Python builtin

(A002)